iotdb-core/datanode/src/main/java/org/apache/iotdb/db/schemaengine/schemaregion/mtree/impl/pbtree/mnode/container/CachedMNodeContainer.java

```diff
@@ -425,14 +416,10 @@ private boolean changeStatus() {
 
   private class BufferIterator implements Iterator<ICachedMNode> {
```

Comment:
It seems this class can be removed since newChildBuffer.iterator can be used directly.

Reply:
fixed